interfaces/grafana_datasource_exchange/v0/interface.yaml

```diff
@@ -0,0 +1,26 @@
+name: grafana_datasource_exchange
+
+internal: true
+
+version: 0
+
+status: draft
+
+providers:
+  - name: tempo-coordinator-k8s
+    url: https://github.com/canonical/tempo-coordinator-k8s-operator
+    branch: grafana_datasource_exchange
```

Comment:
Now that the PR is merged, `branch` can go.

Reply:
unfortunately that branch doesn't exist (yet)

Reply:
working on it

Reply:
Ahh, mistaken it for https://github.com/canonical/tempo-coordinator-k8s-operator/pull/82